drivers/modem/Kconfig.ehs6

```diff
@@ -0,0 +1,65 @@
+# Kconfig - Gemalto EHS6 driver options
+
+#
+# Copyright (c) 2018 Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#CONFIG_OFFLOAD_SOCKET
+
+menuconfig MODEM_EHS6
+	bool "Enable Gemalto EHS6 modem driver"
+	depends on MODEM
+	select MODEM_RECEIVER
+	select NET_OFFLOAD
+	select UART_MCUX_2 if BOARD_FRDM_K64F
+	help
+	  Choose this setting to enable Wistron WNC-M14A2A LTE-M modem driver.
+	  NOTE: Currently the pin settings only work with FRDM K64F shield.
```

Comment:
Is this true for this modem?  Or is this copy/paste issue as well?

Reply:
Sorry, I copied the Wistron driver but forgot to modify the help of Kconfig file.